vagrant-spk

```diff
@@ -162,15 +162,18 @@ GITIGNORE_CONTENTS = r"""
 GLOBAL_SETUP_SCRIPT = r"""#!/bin/bash
 set -euo pipefail
 
+# Set options for curl. Since we only want to show errors from these curl
+# commands, we also use 'cat' to buffer the output; for more information:
+# https://github.com/sandstorm-io/vagrant-spk/issues/158
 CURL_OPTS="--silent --show-error"
 echo localhost > /etc/hostname
 hostname localhost
-curl $CURL_OPTS https://install.sandstorm.io/ > /host-dot-sandstorm/caches/install.sh
+curl $CURL_OPTS https://install.sandstorm.io/ 2>&1 > /host-dot-sandstorm/caches/install.sh | cat
```

Comment:
My test bash heredoc in the pull request demonstrates that this does what I think it does, fwiw.

If you prefer, I can set up some named pipes with `mkfifo` in the script. I experimented with this for a while, and sadly I can't find any other similarly-short invocation that also works.

Let me know what you think.